nidn/fdtd_integration/calculate_transmission_reflection_coefficients.py

```diff
@@ -38,15 +208,14 @@ def calculate_transmission_reflection_coefficients(
                 .item()
             ]
         )
-
     else:
```

Comment:
May want to remove this, as the results could possibly be way off, and rather ask that the user uses more time steps? Will probably be unnecessary with the automatic calculation of niter

Reply:
But we already asking them to increase the number of timesteps, right? Regarding automatic calculation see also https://github.com/esa/NIDN/pull/111/files#r935248046 . I think we should tell the user to do it. Our calculation may be off or the user is just testing something, wants to use fewer timesteps some reason etc. Always forcibly increasing it limits the use options